usagov/usagov-2021
# USAGOV-611-menu-disable-links: Update disableMenuLinks.js

https://cm-jira.usa.gov/browse/USAGOV-611

## Description

Reverting changes so the nav links are still clickable however current page link is now a jump link to h1. 
Also reverts changes of logo so logo always takes you back to home (and reloads if on homepage) 

## Testing instructions 
Navigate to 
http://localhost/admin/structure/block/block-content -> http://localhost/block/11 (English header code) 

In the body code add the following ids: 

1. For the about menu item in "li class.." add: id="usa-nav__about”
2. For the money menu item  in "li class.." add: id="usa-nav__money"
3. For the laws and legal issues menu item  in "li class.." add: id="usa-nav__law"
4. For the scams and fraud menu item  in "li class.. " add: id="usa-nav__scams"

for the Spanish block (http://localhost/block/12) add the following ids: 

1. For the about menu item in "li class.." add: id="usa-nav__acerca”
2. For the money menu item  in "li class.." add: id="usa-nav__dinero"
3. For the laws and legal issues menu item  in "li class.." add: id="usa-nav__leyes"
4. For the scams and fraud menu item  in "li class.. " add: id="usa-nav__estafas"

Clear the cache
Click any top nav link (other than all topics and services) 
Once page loads click link again. The page should not reload but jump to H1. 
Test in both English and Spanish.

On homepage make sure logo goes back to previous functionality (should just reload the homepage) 

## Checklist for the Developer



## Checklist for the Peer Reviewers
